src/backends.jl

```diff
@@ -1,17 +1,50 @@
 abstract type AbstractBackend end
 
 struct JuliaBackend <: AbstractBackend end
-struct CaltechHPC <: AbstractBackend end
+struct SlurmBackend <: AbstractBackend end
+struct CaltechHPC <: SlurmBackend end
+struct ClimaGPU <: SlurmBackend end
 
+"""
+    get_backend()
+
+Get ideal backend for deploying forward model runs.
+
+Each backend is found via specific system information:
+- JuliaBackend: default
+- CaltechHPC: String "Red Hat" in /etc/redhat-release
+- ClimaGPU: Correct GPU names returned from `nvidia-smi --query-gpu=gpu_name --format=csv``
+"""
 function get_backend()
```

Comment:
A better way to identify a backend is using the `hostname`.
`hostname` for `clima` is `clima.gps.caltech.edu`, for `CaltechHPC`, the hostname matches the regular expression `$login[1-4].cm.cluster^`.